package-lock.json

```diff
@@ -6,12 +6,15 @@
     "": {
       "license": "MIT",
       "devDependencies": {
-        "@symfony/webpack-encore": "^1.0.5",
+        "@symfony/webpack-encore": "^1.1.0",
         "bootstrap": "^4.6.0",
         "date-fns": "^2.17.0",
-        "popper.js": "^1.14.3",
-        "sass": "^1.32.6",
-        "sass-loader": "^10.0.0"
+        "eslint-loader": "^4.0.2",
```

Comment:
please don't use the deprecated loader here

Reply:
I was thinking about that but encore [doesn't yet have](https://github.com/symfony/webpack-encore/issues/847) first class support for eslint-webpack-plugin. Would be better to write a custom rule for that now and remove it later on?

Reply:
Eh, it's just eventually replacing two lines of code with one hopefully, let me add that so it makes sense

Reply:
ok, then maybe wait for Encore to support it.

Reply:
I already pushed a new commit that adds the plugin for that, that should continue to work even after encore supports it natively. Gonna be cleanup either way so might as well use a dependency that isn't deprecated.

Reply:
Sorry, i was committing too fast

Reply:
well, as that's simple enough, it is fine with me